lib/Activity/ActivityManager.php

```diff
@@ -410,12 +397,31 @@ private function createEvent($objectType, $entity, $subject, $additionalParams =
 
 		$subjectParams['author'] = $author === null ? $this->userId : $author;
 
+		$subjectParams = array_merge($subjectParams, $additionalParams);
+		$json = json_encode($subjectParams);
+		if (mb_strlen($json) > 4000) {
```

Comment:
Can we use more describable variables? 

Like?

```php
const MAX_SUBJECT_LENGTH = 4000;
const MAX_SUBJECT_CONTENT_LENGTH = 2000;

// ---

if (mb_strlen($json) > self::MAX_SUBJECT_LENGTH) { ...
```

Reply:
Sure :) Added.